packages/valory/skills/termination_abci/models.py

```diff
@@ -38,18 +38,16 @@
 class SharedState(BaseSharedState):
     """Keep the current shared state of the skill."""
 
-    def __init__(self, *args: Any, **kwargs: Any) -> None:
-        """Initialize the state."""
-        super().__init__(*args, abci_app_cls=TerminationAbciApp, **kwargs)
+    abci_app_cls = TerminationAbciApp
 
 
 class TerminationParams(TransactionParams):
     """Defines the class to hold the termination parameters."""
 
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         """Set up the termination parameters."""
-        self.termination_sleep = self._ensure("termination_sleep", kwargs)
-        self.multisend_address = self._ensure("multisend_address", kwargs)
+        self.termination_sleep: int = self._ensure("termination_sleep", kwargs, int)
+        self.multisend_address: str = self._ensure("multisend_address", kwargs, str)
```

Comment:
`multisend_address` will be tricky with apps that make use of it, because of `kwargs.pop()` in `_ensure`.